visual_behavior/utilities.py

```diff
@@ -290,7 +289,7 @@ class Movie(object):
     --------------
     - non-lazy-load a defined interval (would this be useful?)
     '''
-
+    import cv2 # import cv2 here to avoid errors
```

Comment:
Why is this being imported here? If its something thats not always used, maybe make this a separate module, or include the import at the top as a try/except? importing in the class definition seems like bad style (but I dont know what is considered the best style)/

Reply:
Good question. I'm getting a tox error all of the sudden on the cv2 import, but only for the py36 and allensdk_head environments (not py37). Here's the error:

```
visual_behavior/utilities.py:292: in Movie
    import cv2 # import cv2 here to avoid errors
.tox/py36/lib/python3.6/site-packages/cv2/__init__.py:5: in <module>
    from .cv2 import *
E   ImportError: dlopen: cannot load any more object with static TLS
```

This was an attempt to avoid that error by only importing cv2 when the class was instantiated. It didn't work, but even if it had, I'll admit that it was hacky and awful.

I'll have to figure out the root cause. Maybe I need to pin the cv2 version? 

(I hate things like this!!)

Reply:
Alright, I couldn't replicate the openCV issue in a local python 3.6 environment, but pinning openCV to 4.2.0.34 (the April 2020 release) for python<=3.6 solved the issue on circleCI. Tests are passing.

Thanks for calling me out on this attempted hacky fix!